content/pages/healthcare/apply.md

```diff
@@ -66,7 +66,6 @@ To update your personal, insurance, or financial information from a previous app
 ### Additional forms for caregivers, National Guard, and Reserve Servicemembers
```

Comment:
Since we're removing the form for National Guard and Reserve, we should probably change this header. Also the banner at the top of the page.